src/index.html

```diff
@@ -9,6 +9,55 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          class="logo-img"
+          src="./images/logo.png"
+          alt="asd"
+        >
+      </a>
+      <nav class="nav__list">
+        <ul class="ul_list_header">
```

Comment:
The class names 'ul_list_header' and 'li_list_header' are not very descriptive. Consider using more semantic class names. You BEM for a naming classes
<img width="802" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/64750958/eecd8322-7b6a-4fd0-b9e8-0b8153912823">